bukkit/src/main/resources/resources/bukkit/config.yml

```diff
@@ -25,6 +25,22 @@
 
 # Trying to add new categories? This has been moved to categories.yml
 
+#Default section doesn't require any permission
+command:
```

Comment:
This should be with the other messages (though this system is going to redone at some point in the future anyway)